### PR TITLE
Ezp 26835 : use ezplatform i18n to handle translations

### PR DESCRIPTION
**Pull request by nicolas-bastien**
Jira : https://jira.ez.no/browse/EZP-26835

# Description

Translations (other than en) will be stored in ezplatform-i18n repository. 

So this PR : 
- remove empty fr_FR translation files 
- remove updatestring.shas it will be handled in ezplatform
- remove crowdin configuration file as it's not used anymore
- fix HandleBarsFileVisitorTest for translation with variable as translation key
- add extract translation script + update files

### CODE REVIEW

**Comment by bdunogier** [comment]
Could you please find a more meaningful title for the PR ? :)

**Comment by bdunogier** [comment]
Note: we can probably not merge this until the french package is available, can we ?

File: bin/updatestrings.sh [hunk 1]
```diff
@@ -1,7 +1,20 @@
...
+# Extract string for default locale
 ../../../app/console translation:extract en -v \
   --dir=. \
   --exclude-dir=vendor --exclude-dir=Tests --exclude-dir=Features --exclude-dir=node_modules --exclude-dir=Resources/public/vendors \
   --output-dir=./Resources/translations \
   --keep
   "$@"
+
+# Clean corresponding ezplatform-i18n folder
+rm ../../ezsystems/ezplatform-i18n/source/platform-ui-bundle/*.xlf
```
Comment (bdunogier) [inline bin/updatestrings.sh]: Maybe we should output something to say what is being done. The extraction script will output stuff, but the system commands below won't.

**Comment by nicolas-bastien** [comment]
@bdunogier done
So I'm just waiting for https://github.com/ezsystems/ezplatform-i18n to be published on packagist to finish this PR

**Comment by nicolas-bastien** [comment]
@bdunogier @dpobel as discused this morning I remove the updatestring script to implement it on ezplatform directly.

This PR is now just file deletion and a small fix on HandleBarsFileVisitorTest.php

**Comment by nicolas-bastien** [comment]
@bdunogier I added extract script in this PR not to be in conflict

filestypes has been moved to kernel as we discus this morning

**Comment by bdunogier** [comment]
You have a conflict :

**Pull request merged**